### DISCUSSION TITLE
Change to individual entities

### DISCUSSION THREAD

**Discussion by AhoyhoyGit** [muxa/home-assistant-niwa-tides#3] [Ideas]
First off thanks for implementing this! I assumed NZ being so far behind for most things something like this wouldn't exist but was stoked when I saw it did.
At the moment in Home Assistant this shows in a single entity card as 'Local Tides *m' and once you select it and click attributes you can see the full list of available sensors.
Would you be able to make each of them an individual entity\sensor so a custom card can be created having only the sensors you want easy access to directly on screen?

**Comment by muxa** [comment]
That's a good idea. Also HA supports display date/time sensors properly now, so for next tide time etc it would be perfect. 
I have not created a multi-sensor integration before so it will be a bit of a learning curve for me. Unfortunately I don't have a lot of spare time to do this in the next couple of months. 

In the meantime you can use Entity card to display a particular attribute, e.g:
<img width="470" alt="Screen Shot 2022-02-02 at 8 14 19 AM" src="https://user-images.githubusercontent.com/128341/152035343-b78e6552-5bf9-4d6e-bb2f-cf9a473f7536.png">

Or you could create a template sensor. 

A PR to implement this would be welcomed as well :)
